players.md

```diff
@@ -0,0 +1,152 @@
+| **PGSSub**              | ✅                                                                                                             | ❌                                                                           |
+| **SRT**                 | ✅                                                                                                             | ❌                                                                           |
+| **SSA**                 | ✅                                                                                                             | ❌                                                                           |
+| **Teletext**            | ✅                                                                                                             | ❌                                                                           |
+| **TTML**                | ✅                                                                                                             | 🟡                                                                           |
+| **VTT**                 | ✅                                                                                                             | 🟡                                                                           |
+| **XSub**                | ✅                                                                                                             | ❌                                                                           |
+
+## Notes
+
+**Subtitles are not currently supported in the Native Player.** 
+
+---
+
+## HDR Support
+
+| Subtitle Format         | Swiftfin (VLCKit)                                                                                               | Native (AVKit)                                                               |
+|-------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| **Dolby Vision Profile 5**              | ❌                                                                                                             | ✅                                                                           |
+| **Dolby Vision Profile 8**              | ❌                                                                                                             | 🟡 Compatible devices only                                                                         |
+| **Dolby Vision Profile 10**              | ❌                                                                                                             | ✅                                                                           |
```

Comment:
IIRC this is only going to be found with AV1, so support is limited to devices with AV1 hardware decode (iPhone 15 Pro+)

Reply:
Specifically just HDR10 or all HDR profiles?

Reply:
Just Dolby Vision Profile 10 (which is different from HDR10)